src/TrackArt.h

```diff
@@ -21,11 +21,10 @@ namespace TrackArt {
 
    static constexpr int ClipFrameRadius{ 6 };
 
+   //Draws clip affordance and title string, if not empty.
+   //Returns clip's title rectangle
```

Comment:
You might format the comment for Doxygen:
/*!
 @return clip's title rectangle
*/